src/ToricVarieties/ToricDivisorClasses/attributes.jl

```diff
@@ -0,0 +1,73 @@
+@doc Markdown.doc"""
+    class(tdc::ToricDivisorClass)
+
+Return the class group of the toric divisor class `tdc`.
+
+# Examples
+```jldoctest
+julia> P2 = projective_space(NormalToricVariety, 2)
+A normal, non-affine, smooth, projective, gorenstein, fano, 2-dimensional toric variety without torusfactor
+
+julia> tdc = ToricDivisorClass(P2, class_group(P2)([1]))
+A divisor class on a normal toric variety
+
+julia> class(tdc)
+Element of
+GrpAb: Z
+with components [1]
+```
+"""
+function class(tdc::ToricDivisorClass)
+    return tdc.class
+end
```

Comment:
Maybe we want to make this such a one-liner?
```suggestion
class(tdc::ToricDivisorClass) = tdc.class
```

Reply:
Thank you.